instagram/spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+feature "liking photos" do 
+    # {user = User.create}
+    # {user.Photo.create description: "Fujisan"}
+  
+  let (:user) {User.create(email: "test@test.com", 
+                          password: "password123", 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.